public/index.html

```diff
@@ -3,27 +3,46 @@
 	<head>
 		<meta charset="utf-8">
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<link rel="icon" href="favicon.ico">
+		<link rel="icon" href="assets/image/favicon.ico">
 
 		<title>OpenFisca - Turn law into software</title>
 
-		<link rel="stylesheet" type="text/css" href="css/main.css">
+		<link rel="stylesheet" type="text/css" href="assets/css/main.css">
 	</head>
 	<body>
-		<img id="logo" src="logo-openfisca.svg" alt="OpenFisca">
+		<img id="logo" src="assets/image/logo-openfisca.svg" alt="OpenFisca">
 
 		<h1>Turn law into software</h1>
 
 		<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
-
-		<h3>Calculate citizens rights</h3>
-		<p>From an individual situation, calculate any variable of a national tax and benefit system.</p>
-
-		<h3>Predict company taxes</h3>
-		<p>Model any kind of entity and evaluate their eligibility to fiscal obligations and benefits.</p>
-
-		<h3>Assess the impact reforms could have</h3>
-		<p>Compare law as it is now and as it could be.</p>
+		
+		<div class="band">
+			<h3>Calculate citizens' rights</h3>
+			<p>
+				From an individual situation, calculate any variable of a national tax and benefit system.
+				<video class="screenshot" src="assets/video/mes-aides.mp4" poster="assets/image/mes-aides.png" controls></video>
+				<a href="https://beta.gouv.fr/startup/mes-aides">Mes Aides</a> example uses the OpenFisca Web API to calculate OpenFisca-France benefits.
+			</p>
+		</div>
+
+		<div>
+			<h3>Assess company taxes</h3>
+			<p>
+				Model any kind of entity and evaluate their eligibility to fiscal obligations and benefits.
+				<img class="screenshot" src="assets/image/embauche.jpg" alt="embauche.beta.gouv.fr">
+				<a href="https://embauche.beta.gouv.fr/">Embauche</a> application calculates hiring costs.
+			</p>
+		</div>
+
+		<div class="band">
+			<h3>Evaluate reforms impact</h3>
+			<p>
+				Compare law as it is now and as it could be.
+				<img class="screenshot" src="assets/image/revenu-de-base.png" alt="Comment appréhender le coût d’un revenu de base ?">
+				<a href="http://www.revenudebase.info/2017/04/07/apprehender-cout-dun-revenu-de-base/">This publication (in french)</a> 
```

Comment:
What about using a screenshot with more graphics and less text, like http://www.revenudebase.info/wp-content/uploads/2017/04/index01-1024x618.png? 🙂